### PR TITLE
Adds important admin pings, cause I don't trust staff

### PR DESCRIPTION
**Pull request by nichlas0010**
as per usual, yogbot doesn't like my PC so I can't test it

### CODE REVIEW

File: config/discord.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
   "discord_public_channel": "134720091576205312",
   "discord_coder_channel": "134722107190935552",
   "discord_coder_public_channel": "205784753352343552",
+	"discord_important_admin_channel": "134722058688004096",
```
Comment (AshCorr) [inline config/discord.json]: bad

File: models/Byond/Endpoints/EndpointAdminless.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+var APIEndpoint = require('../APIEndpoint.js');
+
+class Endpointadminless extends APIEndpoint {
```
Comment (AshCorr) [inline models/Byond/Endpoints/EndpointAdminless.js]: bad

File: models/Byond/Endpoints/EndpointAdminless.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        status: 400,
+        response: "Missing data parameter."
+      }
+
+      return callback(error, undefined);
+    }
+    var config = this.manager.subsystemManager.getSubsystem("Config").config;
+    var discord = this.manager.subsystemManager.getSubsystem("Discord");
+    for (var channel of discord.getPrimaryGuild().channels.array()) {
+      if (channel.id == discord_important_admin_channel) {
```
Comment (AshCorr) [inline models/Byond/Endpoints/EndpointAdminless.js]: bad

File: config/discord.json [hunk 4]
```diff
@@ -10,7 +10,7 @@
   "discord_public_channel": "134720091576205312",
   "discord_coder_channel": "134722107190935552",
   "discord_coder_public_channel": "205784753352343552",
-	"discord_important_admin_channel": "134722058688004096",
+   "discord_important_admin_channel": "134722058688004096",
```
Comment (AshCorr) [inline config/discord.json]: bad

File: models/Byond/Endpoints/EndpointAdminless.js (hunk 3 above)
Comment (nichlas0010) [reply models/Byond/Endpoints/EndpointAdminless.js]: why is this one bad...

Comment (nichlas0010) [reply models/Byond/Endpoints/EndpointAdminless.js]: I see

**Comment by AshCorr** [comment]
This probably wont work, who cares though!

**Comment by nichlas0010** [comment]
actually works, wow me

**Pull request merged**